src/utils/icon-utils.js

```diff
@@ -0,0 +1,16 @@
+const fs = require("fs");
+
```

Comment:
When this util was out of the src folder it cause issues while creating the dist folder